handlers/register.go

```diff
@@ -34,6 +34,7 @@ func Register(mainRouter *mux.Router, cfg *config.Config, kSvc services.KafkaSer
 
 	appRouter := mainRouter.PathPrefix("").Subrouter()
 	appRouter.HandleFunc("/delta/officers", NewOfficerDeltaHandler(kSvc, h, chv, cfg).ServeHTTP).Methods(http.MethodPost).Name("officer-delta")
+	appRouter.HandleFunc("/delta/officers/validate", NewDeltaValidationHandler(h, chv, cfg).ServeHTTP).Methods(http.MethodPost).Name("officer-delta-validate")
 	appRouter.Use(userAuthInterceptor.UserAuthenticationIntercept)
```

Comment:
By adding it to the appRouter, we are applying the authentication middleware to the route, do we want this route to be protected or should it remain open (like the healthcheck route)? Worth clarifying this.

Reply:
Happy to chase up with Mike Moore. It's a fair point as the validate endpoint isn't pushing any data through to other systems so could potentially not need to be authenticated